unit_testing/myapp/models.py

```diff
@@ -6,5 +7,27 @@ class Person(models.Model):
     first_name = models.TextField(max_length=50)
     last_name = models.TextField(max_length=50)
 
+    cache_key = u'person_{}'
+
     def __unicode__(self):
         return "{name}".format(name=self.first_name+self.last_name)
+
+    def save(self, *args, **kwargs):
+        super(Person, self).save(*args, **kwargs)
+        print("Object is saved")
+
+    def delete(self, *args, **kwargs):
+        cache.delete(self.first_name)
```

Comment:
this is the wrong key. This should be `cache.delete(self.cache_key.format(self.first_name))`